.changeset/config.json

```diff
@@ -5,7 +5,7 @@
   "fixed": [],
   "linked": [],
   "access": "restricted",
-  "baseBranch": "origin/main",
+  "baseBranch": "origin/travelperk",
```

Comment:
I recall seeing a post from the security team the other day stating we should not use other branch names than main

Reply:
Yes, I already talked to them, and forked repos will need to be treated differently.